Clarify that the `name` map should exactly match statement languages. (#103)
Kattis/problem-package-format: spec/2023-07-draft.md

| problem_format_version                  | String                                                      | `legacy`                                                | Version of the Problem Package Format used for this package. If using this version of the Format must be the string `2023-07-draft`. Will be on the form `<yyyy>-<mm>` for a stable version, `<yyyy>-<mm>-draft` or `draft` for a draft version, or `legacy` for the version before the addition of problem_format_version. Documentation for version `<version>` is available at https://www.kattis.com/problem-package-format/spec/problem_package_format/<version>.                                                                                            |

> | name                                    | String or map of strings                                    |                                                         | Required. The name of the problem in the languages that there are problem statements for.

Reviewer: maybe `name(s)`

> If there are statements in more than one language, this must be a map with the language codes as keys and the problem names as values.

Reviewer: I think you're breaking the table layout here? Better move it below the entire table then, similar to `### License`

Author: Done. This will look nicer if (when) we move to Thore's restructuring anyway.

| problem_format_version                  | String                                                      | `legacy`                                                | Version of the Problem Package Format used for this package. If using this version of the Format must be the string `2023-07-draft`. Will be on the form `<yyyy>-<mm>` for a stable version, `<yyyy>-<mm>-draft` or `draft` for a draft version, or `legacy` for the version before the addition of problem_format_version. Documentation for version `<version>` is available at https://www.kattis.com/problem-package-format/spec/problem_package_format/<version>.                                                                                            |

> | name                                    | String or map of strings                                    |                                                         | Required. The names of the problem in the languages that there are problem statements for. See below for details.

Reviewer 2: No ending on a preposition 😄 Better this way?
```suggestion
| name                                    | String or map of strings                                    |                                                         | Required. The name of the problem in each of the languages for which a problem statement exists. See below for details.
```

| <span class="not-icpc">languages</span> | <span class="not-icpc">String or sequence of strings</span> | <span class="not-icpc">all</span>                       | <span class="not-icpc">List of programming languages or the string `all`.</span>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |

### Name
If there are statements in more than one language, the `name` field must be a map with the language codes as keys and the problem names as values.

> The set of languages must **exactly** match that for which a problem statement exists.

Reviewer: ```suggestion
The set of languages for which `name` is given must **exactly** match the set of languages for which a problem statement exists.
```

> If only a single problem statement exists, this may be a string with the names of the problem in that language (but a map with a single key is allowed).

Reviewer: ```suggestion
If only a single problem statement exists, this may be a string with the name of the problem in that language (but a map with a single key is allowed).
```
